src/haz3lschool/SyntaxTest.re

```diff
@@ -30,6 +34,10 @@ let rec find_var_upat = (name: string, upat: Term.UPat.t): bool => {
   };
 };
 
+/*
+ Finds whether a variable with the same name as
+ the argument "name" is ever mentioned in uexp.
+ */
 let rec var_mention = (name: string, uexp: Term.UExp.t): bool => {
```

Comment:
why is the upat version called `find_var_upat` rather than `var_mention_upat`?

Reply:
I think it was because the function was written to be a helper function and I did not change it afterwards